builtin/fstk/ui.lua

```diff
@@ -50,6 +50,18 @@ function ui.find_by_name(name)
 	return ui.childlist[name]
 end
 
+--------------------------------------------------------------------------------
+function ui.get_message_formspec(title, message, btn_id)
+	return table.concat({
+		"size[14,8]",
+		"real_coordinates[true]",
+		"set_focus[", btn_id, ";true]",
+		"box[0.5,1.2;13,5;#000]",
+		("textarea[0.5,1.2;13,5;;%s;%s]"):format(title, message),
```

Comment:
expecting `title` and `message` to be escaped already is intended, right?

Reply:
Yes, it is. I've added some comments.